src/components/VButton.vue

```diff
@@ -72,6 +73,7 @@ const VButton = defineComponent({
           'secondary',
           'tertiary',
           'action-menu',
+          'action-menu-secondary',
```

Comment:
Hmmm, I'm beginning to worry we have too many variants! I wonder if we can ditch any of these? Or maybe nudge @panchovm to unify the button variants so that we don't have so many 🤔

Reply:
You are right. I also noticed I forgot to add the content switcher and filter versions for mobile. I would like to know more about your concern @sarayourfriend, since the unskippable variant I detect is the responsive one. Called Big for desktop elements and **Small** for mobile ones.